js/length-indicator.js

```diff
@@ -52,4 +63,43 @@
     coloredIndicator.addClass('is-active');
   };
 
+  /**
+   * Creates an hidden element with the purpose to calculate the sizes of elements and adds these elements to the body.
+   *
+   * @return {HTMLElement} The created hidden element.
+   */
+  Indicator.prototype.createMeasurementElement = function () {
+    var hiddenElement = document.createElement('div');
+
+    hiddenElement.id = 'length-indicator-measure';
+
+    // Styles to prevent unintended scrolling in Gutenberg.
+    hiddenElement.style.position = 'absolute';
+    hiddenElement.style.left = '-9999em';
+    hiddenElement.style.top = 0;
+    hiddenElement.style.height = 0;
+    hiddenElement.style.overflow = 'hidden';
+    hiddenElement.style.fontFamily = 'arial,sans-serif';
+    hiddenElement.style.fontSize = '18px';
+    hiddenElement.style.fontWeight = '400';
```

Comment:
The values fontFamily, fontSize and fontWeight should be made configurable, see other remark in length_indicator.moduke